### PR TITLE
Turn off the `lib9cStateService` and `remoteActionEvaluatorHeadless`

### PR DESCRIPTION
**Pull request by boscohyun**
SSIA

### CODE REVIEW

**Review by sky1045 — APPROVED** [review]
👍

File: 9c-main/chart/values.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-
```
Comment (moreal) [inline 9c-main/chart/values.yaml:5]: This line change seems not related to this pull request

**Review by moreal — APPROVED** [review]
